app/javascript/packs/public.js

```diff
@@ -39,7 +39,7 @@ function loaded() {
 }
 
 function main() {
-  if (document.readyState === 'complete') {
+  if (['interactive', 'complete'].includes(document.readyState)) {
```

Comment:
Why don't you write `document.readyState !== 'loading'` instead of calling `Array.prototype.includes()`?